tsconfig.json

```diff
@@ -5,9 +5,10 @@
     "noUnusedLocals": true,
     "noUnusedParameters": true,
     "sourceMap": false,
-    "target": "es2017",
+    "target": "es5",
```

Comment:
Does the current lambda runtime not support es6+?

Reply:
ok. Making it 2015

Reply:
I mean es6